### PR TITLE
Start vm immediately on boot

### PR DESCRIPTION
**Pull request by Gamerboy59**
Disable autoboot delay and logo display

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -98,10 +98,12 @@ touch /etc/rc.conf
     echo 'boot_multicons="YES"' >> /mnt/boot/loader.conf
     echo 'boot_serial="YES"' >> /mnt/boot/loader.conf
     echo 'comconsole_speed="115200"' >> /mnt/boot/loader.conf
-    echo 'autoboot_delay="1"' >> /mnt/boot/loader.conf
+    echo 'autoboot_delay="-1"' >> /mnt/boot/loader.conf
```
Comment (goneri) [inline build.sh:101]: How do you do if you need to recover a broken system and jump back on an older kernel?

Comment (Gamerboy59) [reply build.sh:101]: The docs say: `If set to "-1", no delay will be inserted and loader.conf starts interactive mode only if autoboot has failed.`  
My assumption is that if you need to recover a broken system, you can still access interactive mode.  
Prioritising a kernel for next reboot can be done in upfront via `grub.cfg` or `loader.conf` iirc.

**Comment by Gamerboy59** [comment]
FreeBSD14.2 has been released. Any chance the PR is being accepted for it?

**Comment by goneri** [comment]
Thank you for the clarification.

**Pull request merged**